src/main/java/lotto/constant/LottoConstant.java

```diff
@@ -0,0 +1,22 @@
+package lotto.constant;
+
+public enum LottoConstant {
+    NUMBER_RANGE_MIN(1),
+    NUMBER_RANGE_MAX(45),
```

Comment:
```suggestion
    MIN_NUMBER_RANGE(1),
    MAX_NUMBER_RANGE(45),
```
사소한 건데 이런 네이밍이 더 좋은 것 같아요!

Reply:
읽으면서 느낍니다.. 확실히 더 좋은 네이밍 의견입니다. 감사합니다.